app/helpers/loss_report_helper.rb

```diff
@@ -0,0 +1,189 @@
+module LossReportHelper
+
+  def cmr_client
+    Cmr::Client.client_for_environment(Rails.configuration.cmr_env, Rails.configuration.services)
+  end
+
+  def token
+    if session[:login_method] == 'launchpad'
+      session[:launchpad_cookie]
+    elsif session[:login_method] == 'urs'
+      session[:access_token]
+    end
+  end
+
+  def loss_report_output(hide_items=true, disp='text')
+    # depending on the input selection (json or text) a comparison string/hash is created and displayed in-browser
+
+    # @collection_error is true if there is an error in the translation that is performed by prepare_collections in the collections_controller
```

Comment:
I think you need to add that there error is not something that the method is doing wrong, rather there was something wrong with what CMR gave us back